content/en/docs/ops/configuration/mesh/config-resource-ready/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Wait for Resource Status to Apply Configuration
-description: Describes how to wait to apply mesh configuration until a resource reaches a given status or readiness.
+title: Wait on Resource Status for Applied Configuration
+description: Describes how to wait until a resource reaches a given status or readiness.
```

Comment:
```suggestion
description: Describes how to wait until a resource reaches a given status of readiness.
```

Reply:
@frankbu I fixed this and did the `/retest` but something is failing unrelated to this change.  Thoughts?